src/html/ui/community/settings.tpl.html

```diff
@@ -213,5 +213,29 @@
 
     </div>
 
+    <div class="section-label" ng-click="expand5 = !expand5">
+      Send Updates to Slack
+      <i ng-class="{'icon-down': expand5, 'icon-right': !expand5}"></i>
+    </div>
+    <div class="section slack" collapse="!expand5">
+      <div class="setting-item">
+        <div class="full-column">
+          <div class="add-to-slack" ng-hide="community.slack_hook || slack_remove_transition">
+            <p>Connect this community to a <a href="https://slack.com" target="_blank">Slack</a> team and Hylo will notify a channel when there are new posts.</p>
+            <a href="https://slack.com/oauth/authorize?scope=incoming-webhook&client_id=15397411765.20785715792&redirect_uri={{add_slack_url}}"><img alt="Add to Slack" height="40" width="139" src="https://platform.slack-edge.com/img/add_to_slack.png" srcset="https://platform.slack-edge.com/img/add_to_slack.png 1x, https://platform.slack-edge.com/img/add_to_slack@2x.png 2x"></a>
```

Comment:
Actually @levity I'm a little bit lost as to how to actually get an environment variable into this strign